src/LongRunning/OperationsClient.php

```diff
@@ -435,7 +478,11 @@ public function cancelOperation($name, $optionalArgs = [])
      * Sample code:
      * ```
      * try {
-     *     $operationsClient = new OperationsClient();
+     *     $options = [
+     *         'serviceAddress' => 'my-service-address',
+     *         'scopes' => ['my-service-scope'],
+     *     ];
+     *     $operationsClient = new OperationsClient($options);
```

Comment:
Yes - this refresh of the longrunning client is updating the sample code to reflect that you need to pass in the serviceAddress and scopes manually if constructing an OperationsClient (usually you would not construct an OperationsClient yourself - one is created and held by API clients with longrunning operations)

This is independent of the update to the protobuf implementation - I will refresh the operations client separately.